AdminCore.Services.Tests/ContractServiceTests.cs

```diff
@@ -161,32 +182,44 @@ public void TestSaveContractAttemptsUpdateIfContractToBeSavedHasIdOfNotZeroAndCo
     public void TestSaveContractThrowsExceptionIfContractToBeSavedHasIdOfNotZeroAndContractDoesNotExist()
     {
       var newContractDto = BuildContractDto();
-      Assert.Throws<Exception>(() => _contractService.SaveContract(newContractDto));      
+      Assert.Throws<Exception>(() => _contractService.SaveContract(newContractDto));
     }
 
     private static Contract BuildContractModel()
     {
-      return new Contract()
+      return new Contract
       {
         ContractId = 1,
-        EmployeeId = 1,
         TeamId = 1,
-        Team = new Team()
-        {
-          TeamId = 1
-        },
+        EmployeeId = 1,
         StartDate = new DateTime(2018, 12, 10),
-        EndDate = new DateTime(2019, 1, 10)
+        EndDate = new DateTime(2019, 1, 10),
+        Team = new Team
```

Comment:
I feel like objects like this should be mocked, `Team` object changes and your tests break.

Reply:
Yeah I agree, I tried to fix the already existing tests while keeping the functionality the same so I didn't make any big changes to them.

Reply:
Maybe instead, we should consider adding a tech debt task that will fix all these tests.